core/src/main/java/jenkins/slaves/JnlpSlaveAgentProtocol.java

```diff
@@ -42,56 +42,20 @@
  * @author Kohsuke Kawaguchi
  * @since 1.467
  */
-@Extension
-@Symbol("jnlp")
 public class JnlpSlaveAgentProtocol extends AgentProtocol {
```

Comment:
Why not just delete it?

Reply:
But maybe that could be moved to `JnlpAgentReceiver`, the field here aliased to that one, and then this whole class deprecated?